CONTRIBUTING.md

```diff
@@ -707,265 +707,7 @@ Verify the config by calling `bin/configlet .` (notice the dot). This says
-helpful:
-
-* [Git for Ages 4 and Up](https://www.youtube.com/watch?v=1ffBJ4sVUb4) - video of a presentation/demo
-* [Think Like a Git](http://think-like-a-git.net/)
-* [Explain Git with D3](http://www.wei-wang.com/ExplainGitWithD3) - interactive site
-* [Pro Git](http://git-scm.com/book/en/v2) - "The Book" for learning Git in detail
-* [Git Branching Tutorial](http://pcottle.github.io/learnGitBranching/) - interactive tutorial, very visual
-
-#### Rebasing
-
-You'll often be asked to rebase your branch before we merge a pull request as Exercism likes to keep a linear project commit history. This is accomplished with git rebase. It takes the current branch and places all the commits at the front of the branch that you're rebasing with.
-
-For example, rebasing the current branch on top of upstream/master:
-```
- git rebase upstream/master
-```
-Project commit history:
-```
-                       -- current branch --
-                      /
---- master branch ----
-```
-
-##### Interactive Rebase
-The rebase command has an option called `-i, --interactive` which will open an editor with a list of the commits which are about to be changed. This list accepts commands, allowing the user to edit the list before initiating the rebase action.
-
-Using the example from above, we have 5 commits that should be squashed into one.
-
-```bash
-433a0e1 added spinning wheel tests
-1f7d4aa pass spinning wheel
-cf21737 oops
-be4c410 rename example file
-bb89a77 update config
-```
-
-To interactively rebase, use the following incantation:
-
-```bash
-$ git rebase -i HEAD~5
-```
-This will bring up an editor with the following information:
-
-```bash
-pick 433a0e1 added spinning wheel tests
-pick 1f7d4aa pass spinning wheel
-pick cf21737 oops
-pick be4c410 rename example file
-pick bb89a77 update config
-
-#
-# Commands:
-#  p, pick = use commit
-#  r, reword = use commit, but edit the commit message
-#  e, edit = use commit, but stop for amending
-#  s, squash = use commit, but meld into previous commit
-#  f, fixup = like "squash", but discard this commit's log message
-#  x, exec = run command (the rest of the line) using shell
-#
-# These lines can be re-ordered; they are executed from top to bottom.
-#
-# If you remove a line here THAT COMMIT WILL BE LOST.
-#
-# However, if you remove everything, the rebase will be aborted.
-#
-# Note that empty commits are commented out
-```
-
-By choosing the `reword` command for the top commit and choosing the `fixup` command for the remaining commits, you will be able to squash the commits into one commit and provide a descriptive summary of the entire change
-
-```bash
-reword 433a0e1 added spinning wheel tests
-fixup 1f7d4aa pass spinning wheel
-fixup cf21737 oops
-fixup be4c410 rename example file
-fixup bb89a77 update config
-```
-
-[Further Reading](https://www.atlassian.com/git/tutorials/merging-vs-rebasing)
+Refer to [documentation](https://github.com/exercism/docs/tree/master/contributing-to-language-tracks).
```

Comment:
* "documentation" is a bit vague for link text.
* Should the url point to git basics?